bindings/rust/s2n-tls/Cargo.toml

```diff
@@ -6,6 +6,9 @@ authors = ["AWS s2n"]
 edition = "2021"
 repository = "https://github.com/aws/s2n-tls"
 license = "Apache-2.0"
+include = [
+  "src"
```

Comment:
https://doc.rust-lang.org/cargo/reference/manifest.html#the-exclude-and-include-fields
> If include is not specified, then the following files will be excluded:
>
>   - If the package is not in a git repository, all "hidden" files starting with a dot will be skipped.
>   - If the package is in a git repository, any files that are ignored by the [gitignore](https://git-scm.com/docs/gitignore) rules of the repository and global git configuration will be skipped.

i dont think src is needed. you could try and include the pem file from the previous dir.. but honestly copying it into testing is not a huge deal imo

Reply:
Without src added to include, the pems directory was not included in the crate. I'm not sure why, maybe because there's no .rs files in it? I didn't want to include it in an existing directory because we couldn't delete it beforehand like we do with the other copied files in generate.sh. If I only add testing/pems to include, it excludes all other directories... 

I tried adding the original pems in tests/pems, but it doesn't seem to like that. I think they need to be inside the crate directory for it to find them.